src/NuGet.Services.SearchService/Web.config

```diff
@@ -49,23 +49,23 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.Runtime" publicKeyToken="B03F5F7F11D50A3A" culture="neutral"/>
-        <bindingRedirect oldVersion="0.0.0.0-4.1.1.0" newVersion="4.1.1.0"/>
+        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.1.2.0"/>
```

Comment:
Consider regenerating from scratch to ensure it is minimal, as mentioned above with BasicSearch.

Reply:
regenerated